Dockerfile.buster

```diff
@@ -54,8 +54,13 @@ RUN \
   && echo "deb-src https://deb.nodesource.com/${NODE_VERSION} buster main" | tee -a /etc/apt/sources.list.d/nodesource.list \
   && apt-get update \
   && apt-get install -y --no-install-recommends nodejs \
+  && apt-get install -y mariadb-server \
+  && curl -sS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | bash \
+  && apt-get install -y libmariadb3 \
```

Comment:
Do we need both of these dependencies to run the client?

Reply:
We may not need `&& curl -sS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | bash \`  but I'm removing all of these now because the work for the Dockerfile is being done here  #2584